packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -0,0 +1,125 @@
+@import '../../../global/global.pcss';
+
+[part='container'] {
+  box-shadow: 0px 4px 8px rgba(4, 8, 31, 0.08), inset 0px 0px 0px 1px #e5e8e8;
+  margin: 0 auto 30px auto;
+}
+
+[part='header-label'] {
+  border-radius: 8px;
+  color: #1372ec;
+  padding: 8px 12px;
+  font-weight: 500;
+  display: inline-block;
+}
+
+[part='generated-text'] {
+  margin-bottom: 0;
+  margin-top: 20px;
+  color: var(--atomic-neutral-dark);
```

Comment:
We usually use the `--atomic-on-background` color for text, we should probably use it here too.